### PR TITLE
tensorflow bincount function added

### PR DESCRIPTION
**Pull request by adityarajsahu**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by DragosStoican** [comment]
Hi @adityarajsahu , can you pleas link the issue that this PR is trying to solve? Like explained in this section of the docs [here](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues) . Thanks!

**Comment by DragosStoican** [comment]
Also, please address the formatting errors created by your modifications [here](https://github.com/unifyai/ivy/actions/runs/5516566856/jobs/10058076664?pr=19199) .

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -101,6 +101,44 @@ def test_tensorflow_add(
...
+    frontend,
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:135]: missing comma

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 2]
```diff
@@ -101,6 +101,44 @@ def test_tensorflow_add(
...
+        x=x[0]
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:136]: missing comma

**Comment by adityarajsahu** [comment]
@DragosStoican on testing locally, I am getting the error: AttributeError: 'NoneType' object has no attribute 'backend'.

**Pull request closed without merging**